crates/sui-protocol-config/src/lib.rs

```diff
@@ -1614,6 +1621,11 @@ impl ProtocolConfig {
             .record_consensus_determined_version_assignments_in_prologue
     }
 
+    pub fn record_additional_state_digest_in_prologue(&self) -> bool {
+        self.feature_flags
```

Comment:
worth asserting somewhere that if this is true, the earlier prologue flags must also be true?

Reply:
i think its the other way around - i have an assert later on that this is set if we are using a >0 window size. But this can be on by itself and nothing will break.